database/skylink.go

```diff
@@ -118,5 +117,7 @@ func ExtractSkylinkHash(skylink string) (string, error) {
 
 // ValidSkylinkHash returns true if the given string is a valid skylink hash.
```

Comment:
Is the comment here outdated? It talks about a hash but we are actually just loading a skylink.

Reply:
It's incorrectly worded - it was always meant to say "skylink". I'll change it in the next PR.